5.4.kalim_shlovim.py

```diff
@@ -0,0 +1,44 @@
+SPACE = ' '
+
+
+def interleaveGenerator(*args):
+    """ accepts one or more iterable parameters, and returns a list of the interwoven members.
+    :param args: one or more iterable parameters
+    :return: list of the interwoven elements
+    """
+    args = get_list_of_lists(args)
+    for i in range(len(args[0])):
+        yield from [arg[i] for arg in args]
+
+
+def get_list_of_lists(args):
+    """ check if all in same size and convert to lists
+    :param args:
+    :return: list of lists of the arguments
+    """
+    # //check if all in same size
+    size = len(args[0])
+    assert all(len(arg) == size for arg in args), "All arguments must be the same size"
```

Comment:
Dont use assertions to validate user input - 
https://wiki.python.org/moin/UsingAssertionsEffectively